app/DoctrineMigrations/Version20190708092901.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190708092901 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE restaurant ADD pledge_id INT DEFAULT NULL');
```

Comment:
We shouldn't need to add a column to both tables. A `restaurant_id` column in the `restaurant_pledge` table should be enough. I think you need to add `mappedBy` & `inversedBy` in `Restaurant.Pledge.orm.yml` & `Restaurant.orm.yml`. Check [Doctrine docs](https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/association-mapping.html#one-to-one-bidirectional), it only adds a column in one table.